### ISSUE TITLE
Crash on start

### ISSUE THREAD

**Issue by Ramzec** [coreos/torus#242]
I tried to run this changeset
```
commit 59c2a2d205dea622c47fe01e88d8798043a2f259
Merge: 963fe55 80899dc
Author: Barak Michener <me@barakmich.com>
Date:   Mon Jun 6 15:04:20 2016 -0700

    Merge pull request #236 from barakmich/bump
    
    Bump raw to support BSD, Fix #231
```

```
./bin/torusd --etcd 127.0.0.1:2379 --debug --debug-init --peer-address 127.0.0.1:40000

2016-06-07 21:14:21.952972 D | torus: running InitMDS for service type: etcd
debug-init: Already exists
2016-06-07 21:14:21.959815 I | torus: creating metadata service: etcd
2016-06-07 21:14:21.971329 I | torus: creating blockstore: mfile
2016-06-07 21:14:21.972191 I | storage: loading block index...
2016-06-07 21:14:21.972539 D | storage: index memory usage: 0K
2016-06-07 21:14:21.972880 I | storage: done loading block index
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x66eb75]

goroutine 1 [running]:
panic(0xc06d80, 0xc82000e110)
        /usr/lib/go-1.6/src/runtime/panic.go:464 +0x3e6
github.com/coreos/torus/distributor/protocols.ListenRPC(0xc82028a400, 0x7f42e4205100, 0xc82027a870, 0x80000, 0xc82029c7e0, 0x2, 0x4, 0x2
, 0x0, 0x0, ...)
        /home/roman/go/src/github.com/coreos/torus/distributor/protocols/protocols.go:45 +0xd5
github.com/coreos/torus/distributor.newDistributor(0xc820100000, 0xc82028a400, 0x47da51, 0x0, 0x0)
        /home/roman/go/src/github.com/coreos/torus/distributor/distributor.go:45 +0x24d
github.com/coreos/torus/distributor.openReplication(0xc820100000, 0xc82028a400, 0x0, 0x0)
        /home/roman/go/src/github.com/coreos/torus/distributor/replication.go:28 +0x7f
github.com/coreos/torus/distributor.ListenReplication(0xc820100000, 0xc82028a400, 0x0, 0x0)
        /home/roman/go/src/github.com/coreos/torus/distributor/replication.go:16 +0x37
main.runServer(0x1170400, 0xc8201feba0, 0x0, 0x6)
        /home/roman/go/src/github.com/coreos/torus/cmd/torusd/main.go:204 +0x5d2
github.com/coreos/torus/vendor/github.com/spf13/cobra.(*Command).execute(0x1170400, 0xc82000a0f0, 0x6, 0x6, 0x0, 0x0)
        /home/roman/go/src/github.com/coreos/torus/vendor/github.com/spf13/cobra/command.go:565 +0x85a
github.com/coreos/torus/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x1170400, 0x1170400, 0x0, 0x0)
        /home/roman/go/src/github.com/coreos/torus/vendor/github.com/spf13/cobra/command.go:651 +0x55c
github.com/coreos/torus/vendor/github.com/spf13/cobra.(*Command).Execute(0x1170400, 0x0, 0x0)
        /home/roman/go/src/github.com/coreos/torus/vendor/github.com/spf13/cobra/command.go:610 +0x2d
main.main()
        /home/roman/go/src/github.com/coreos/torus/cmd/torusd/main.go:75 +0x2f
```

**Comment by Ramzec** [comment]
I added a debug to function RegisterRPCListener() to be able to see listeners, but i didn't see anyone who registered.

**Comment by Ramzec** [comment]
It seems I have found the root issue. The Makefile contains little incorrect command to run torusd.

instead of
```
./bin/torusd --etcd 127.0.0.1:2379 --debug --debug-init --peer-address 127.0.0.1:40000
```
must be
```
./bin/torusd --etcd 127.0.0.1:2379 --debug --debug-init --peer-address http://127.0.0.1:40000
```
or 
```
./bin/torusd --etcd 127.0.0.1:2379 --debug --debug-init --peer-address tdp://127.0.0.1:40000
```

**Comment by Ramzec** [comment]
https://github.com/coreos/torus/pull/259 fixes the crash

**Comment by barakmich** [comment]
Yep, that was the issue; Good find! Fixed in #259
